src/server/report/test.js

```diff
@@ -0,0 +1,159 @@
+import './test-result.js';
+import { css, html, LitElement, nothing } from 'lit';
+import { FULL_MODE, getId, LAYOUTS } from './common.js';
+import data from './data.js';
+
+class Test extends LitElement {
+	static properties = {
+		file: { type: String },
+		fullMode: { attribute: 'full-mode', type: String },
+		layout: { type: String },
+		showOverlay: { attribute: 'show-overlay', type: Boolean },
+		test: { type: String },
+	};
+	static styles = [css`
+		.header {
+			background-color: #f0f0f0;
+			border-bottom: 1px solid #e6e6e6;
+			box-shadow: 0 0 6px rgba(0,0,0,.07);
+			position: sticky;
+			top: 0;
+			z-index: 2;
+		}
+		.title h2 {
+			margin: 0;
+		}
+		.settings {
+			align-items: center;
+			display: flex;
+			padding-top: 20px;
+			gap: 20px;
+		}
+		.settings-box {
+			background-color: #ffffff;
+			border: 1px solid #cdd5dc;
+			border-radius: 5px;
+			line-height: 24px;
+			padding: 10px;
+			user-select: none;
+		}
+		.header, .results {
+			padding: 20px;
+		}
+		.pill-box {
+			border-radius: 5px;
+			display: flex;
+			flex-wrap: nowrap;
+		}
+		.pill input[type="radio"] {
+			position: absolute;
+			opacity: 0;
+			pointer-events: none;
+		}
+		.pill label {
+			align-items: center;
+			background-color: #ffffff;
+			border-block-end-style: solid;
+			border-block-start-style: solid;
+			border-inline-start-style: solid;
+			border-color: #cdd5dc;
+			border-width: 1px;
+			cursor: pointer;
+			display: flex;
+			gap: 5px;
+			line-height: 24px;
+			padding: 10px;
+			position: relative;
+			user-select: none;
+		}
+		.pill:first-child label {
+			border-start-start-radius: 5px;
+			border-end-start-radius: 5px;
+		}
+		.pill:last-child label {
+			border-end-end-radius: 5px;
+			border-inline-end-style: solid;
+			border-start-end-radius: 5px;
+		}
+		.pill input[type="radio"]:checked + label {
+			background-color: #007bff;
+			color: white;
+		}
+		.pill input[type="radio"]:focus-visible + label {
+			z-index: 1;
```

Comment:
This was needed otherwise the right side of the outline goes under the next item. If anyone has alternate ideas...